### PR TITLE
Removed Amar

### PR DESCRIPTION
**Pull request by lgodolphin**

### CODE REVIEW

File: src/components/about/employees.js [hunk 1]
```diff
@@ -57,18 +55,12 @@ import matthijs from '../../static/employees/DSC01869.jpg'
 import matthijs_ from '../../static/employees/DSC01875.jpg'
 import max from '../../static/employees/DSC01825.jpg'
 import max_ from '../../static/employees/DSC01829.jpg'
-import meriem from '../../static/employees/DSC01783.jpg'
```
Comment (m-mcaulay) [inline src/components/about/employees.js]: do we also need to remove the corresponding images? @kdamball @lgodolphin

Comment (kdamball) [reply src/components/about/employees.js]: Nice to do that too but not the end of the world if some images are left floating around 😅.

**Comment by m-mcaulay** [comment]
qa: http://staging.honeypot.co/pr-62/about/

**Comment by Fennebh** [comment]
@m-mcaulay i don't see amar!

**Pull request merged**